components/style/properties/gecko.mako.rs

```diff
@@ -1964,6 +1965,21 @@ fn static_assert() {
         self.gecko.mTextEmphasisStyle = other.gecko.mTextEmphasisStyle;
     }
 
+    <%call expr="impl_color_setter('_webkit_text_fill_color', 'mWebkitTextFillColor')"></%call>
+    <%call expr="impl_color_copy('_webkit_text_fill_color', 'mWebkitTextFillColor')"></%call>
+    <%call expr="impl_color_clone('_webkit_text_fill_color', 'mWebkitTextFillColor')"></%call>
+
+    <%call expr="impl_color_setter('_webkit_text_stroke_color', 'mWebkitTextStrokeColor')"></%call>
+    <%call expr="impl_color_copy('_webkit_text_stroke_color', 'mWebkitTextStrokeColor')"></%call>
+    <%call expr="impl_color_clone('_webkit_text_stroke_color', 'mWebkitTextStrokeColor')"></%call>
```

Comment:
I made the fix like following:
```
${helpers.predefined_type(
    "-webkit-text-stroke-color", "CSSColor",
    "CSSParserColor::CurrentColor",
    products="gecko", gecko_ffi_name="mWebkitTextStrokeColor",
    complex_color=True, animatable=True, need_clone=True)}
```
and remove the impl_color_{setter, copy, clone} in gecko.mako.rs. Then, I can't build stylo anymore with following error messages:
```
no method named `set__webkit_text_stroke_color` found for type `&mut gecko_properties::GeckoInheritedText` in the current scope
no method named `copy__webkit_text_stroke_color` found for type `&mut gecko_properties::GeckoInheritedText` in the current scope
no method named `clone__webkit_text_stroke_color` found for type `&mut gecko_properties::GeckoInheritedText` in the current scope
```
Did I miss something?